rules/gfb.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var message = require('lambda-cfn').message;
+var util = require('util');
+
+module.exports.config = {
+    name: 'gfb',
+    sourcePath: 'rules/gfb.js',
+    gatewayRule: {
+        method: 'POST',
+        apiKey: false
+    }
+};
+
+module.exports.fn = function (event, callback) {
+    if (event.zen != undefined) {
+        var ping = 'GitHub ping event received';
+        return callback(null, ping);
+    } else {
+        console.log('event ', event);
+        if (event.action === 'labeled'){
+            var label = event.label;
+            if (label.name === 'good-starter-bug') {
```

Comment:
This label name is hard coded as `good-starter-bug` - we'll need to rewrite this so someone passes in the label name as a parameter. This will allow people to use this rule for labels other than `good-starter-bug`.